### PR TITLE
#162633481 Implements Comment History

### PR DESCRIPTION
**Pull request by Qadriyah**
**What does this PR do?**

Implements tracking comment history

**Description of Task to be completed?**

- Creates a commentHistory component
- Displays the component below each comment
- Creates a button to show or hide the component
- Write tests for the feature

**How should this be manually tested?**

- Open a particular article
- Comment on the article
- Click the comment history button

**Any background context you want to provide?**

**What are the relevant pivotal tracker stories?**

#162633481('https://www.pivotaltracker.com/n/projects/2231917/stories/162633481')

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "test:w": "jest --watch --coverage",
     "test:coverage": "jest --coverage --colors",
     "eject": "react-scripts eject",
-    "start": "node server.js",
+    "start": "webpack-dev-server --mode development --open --hot",
```
Comment (jeanjoe) [inline package.json]: Please use the `node server.js` instead

**Review by jeanjoe — DISMISSED** [review]

**Comment by jeanjoe** [comment]
LGTM

**Comment by ruganda** [comment]
Resolve the conflicts @Qadriyah

**Comment by fatukunda** [comment]
LGTM

**Pull request merged**